jobs/transforms/base_model_audits_test.py

```diff
@@ -383,3 +384,85 @@ def test_validate_commit_type(self):
         self.assert_pcoll_equal(output, [
             audit_errors.InvalidCommitTypeError(invalid_commit_type_model),
         ])
+
+
+class MockValidateCommitCmdsSchemaChangeDomain(
+        base_model_audits.ValidateCommitCmdsSchema):
+
+    def _get_change_domain_class(self, item):
+        pass
+
+
+class MockValidateWrongSchema(base_model_audits.ValidateCommitCmdsSchema):
+
+    def _get_change_domain_class(self, item): # pylint: disable=unused-argument
+        return change_domain.BaseChange
+
+
+class ValidateCommitCmdsSchemaTests(job_test_utils.PipelinedTestBase):
```

Comment:
Please add a testcase, where there is no errors also.

Reply:
Done! @rohitkatlaa PTAL!